src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/markup/ContentBuilder.java

```diff
@@ -65,8 +65,8 @@ public ContentBuilder add(CharSequence text) {
             ensureMutableContents();
             Content c = contents.isEmpty() ? null : contents.get(contents.size() - 1);
             TextBuilder tb;
-            if (c != null && c instanceof TextBuilder) {
-                tb = (TextBuilder) c;
+            if (c instanceof TextBuilder txtBldr) {
+                tb = txtBldr;
             } else {
                 contents.add(tb = new TextBuilder());
             }
```

Comment:
Restructure this
```
contents.add((c instanceof TextBuilder tb) ? tb : new TextBuilder());
```
or something similar

Reply:
Unclear if there's a more succinct way to refactor this like so. If `c` is not an instance of `TextBuilder` then we need to initialize a new `TextBuilder` and add it to `contents`. This is not the case if `c` is already of the desired instance.

Reply:
Then just a short name than `txtBldr` please. Lvng t vwls s nt - jvdc cnvntn.

Reply:
I guess we have competing demands for short names!